qutip/qip/device/processor.py

```diff
@@ -700,7 +705,7 @@ def run_state(self, init_state=None, analytical=False, states=None,
 
         evo_result = solver(
             H=noisy_qobjevo, rho0=init_state,
-            tlist=noisy_qobjevo.tlist, **kwargs)
+            tlist=self.tlist, **kwargs)
```

Comment:
See comment in `.get_qobjevo` about calculating the full tlist in its own method rather than storing it on self.